src/frontend/src/pages/DesignReviewSummaryModal.tsx

```diff
@@ -0,0 +1,186 @@
+import { DesignReview, DesignReviewStatus } from 'shared';
+import NERModal from '../components/NERModal';
+import { Box, Button, FormControlLabel, Grid, IconButton, Typography } from '@mui/material';
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import DescriptionIcon from '@mui/icons-material/Description';
+import VideocamIcon from '@mui/icons-material/Videocam';
+import EditIcon from '@mui/icons-material/Edit';
+import { meetingTimePipe } from '../utils/pipes';
+import Checkbox from '@mui/material/Checkbox';
+import { useState } from 'react';
+import { StageGateModal } from '../components/DesignReviewStageGateModal';
+import { DesignReviewDelayModal } from '../components/DesignReviewDelayModal';
+import { DesignReviewPill } from '../components/DesignReviewPill';
+import { DesignReviewMemberPill } from '../components/DesignReviewMemberPill';
+
+interface DRCSummaryModalProps {
+  open: boolean;
+  onHide: () => void;
+  designReview: DesignReview;
+}
+
+const DRCSummaryModal: React.FC<DRCSummaryModalProps> = ({ open, onHide, designReview }) => {
+  const [checked, setChecked] = useState<boolean>(false);
+  const [showStageGateModal, setShowStageGateModal] = useState<boolean>(false);
+  const [showDelayModal, setShowDelayModal] = useState<boolean>(false);
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setChecked(event.target.checked);
+  };
+
+  return (
+    <NERModal open={open} onHide={onHide} title={designReview.wbsName} hideFormButtons>
+      <StageGateModal
+        wbsNum={designReview.wbsNum}
+        modalShow={showStageGateModal}
+        handleClose={() => setShowStageGateModal(false)}
+      />
+      <DesignReviewDelayModal open={showDelayModal} onHide={() => setShowDelayModal(false)} dr={designReview} />
+      <IconButton sx={{ position: 'absolute', right: 16, top: 12 }}>
+        <EditIcon />
+      </IconButton>
+
+      <Grid container direction="column">
```

Comment:
where u can I would split this code up into sections that you put into helperComponents in this file above the return statement. It'll abstract sections of the code so it's easier to see what's going on and it'll make looking at the overall structure of the component easier as well